docs/source/macos/nodejs.rst

```diff
@@ -1,115 +1,209 @@
 
-   export N_PREFIX=~/"n"
-   export PATH=~/"n/bin:$PATH"
 
-Before starting with the next step, restart your terminal app to make
-sure that the path changes are picked up by the shell.
+Troubleshooting
+~~~~~~~~~~~~~~~
 
-3 Install Node.js through ``n``
--------------------------------
+If you are getting ``nvm: command not found`` after the installation, it is
+probably related to the shell you are using. The first step would be to restart
+your terminal or run ``. ~/.nvm/nvm.sh``.
+If this does not help, go through the steps at https://github.com/nvm-sh/nvm#troubleshooting-on-macos.
 
-You can now use ``n`` to install Node.js, for example latest and LTS
+
+2 Install Node.js through ``nvm``
+---------------------------------
+
+You can now use ``nvm`` to install Node.js, for example the latest and LTS
 versions:
 
 .. code:: console
 
-   n latest
-   n lts
+   nvm install node # "node" is an alias for the latest version
+   nvm install --lts
 
-To install exact version of Node.js, for example ``10.16.0``, execute on
+To install a specific version of Node.js, for example ``19.7.0``, execute on
 the command line:
 
 .. code:: console
 
-   n 10.16.0
+   nvm install 19.7.0
 
-You can also install the latest release of the specific major version,
+You can also install the latest release of a specific major version,
 for example ``12``, with:
 
 .. code:: console
 
-   n 12
+   nvm install 12
 
-Note that each version on Node.js installed through ``n`` will come with
+Note that each version of Node.js installed through ``nvm`` will come with
 its own version of ``npm``.
 
-4 Install a package with ``npm``
---------------------------------
 
-To install a package globally, for example ``yarn``, execute on the
-command line:
+3 Switch between different versions of Node.js
+----------------------------------------------
+
+To switch between different versions of Node.js, execute on the command
+line:
 
 .. code:: console
 
-   npm install -g yarn
+   nvm list
 
-Packages installed globally with ``npm`` will be installed independently
-of the version of Node.js that is currently active.
+This will show you the currently installed versions of Node.js on your system.
+You can select one of those, or install another one, and then execute the
+following:
 
-5 Switch between different versions of Node.js
-----------------------------------------------
+.. code:: console
 
-To switch between different versions of Node.js, execute on the command
-line:
+   nvm use 19.7.0
+
+However, you will rarely need to do that - usually, your project should have
+a ``.nvmrc`` file, which already has the required Node.js version defined.
+In that case, all you need to do is run:
 
 .. code:: console
 
-   n
+   nvm use
+
+This command, however, will fail if the requested version of Node.js is not
+already installed, and will require you to install it first.
+To get around this, you can run:
+
+.. code:: console
+
+   nvm use || nvm install $(cat .nvmrc)
```

Comment:
Tested that this `cat` part is not really necessary, it automatically reads the .nvmrc. Unless it's some sort of security for using the alias in directories with no .nvmrc file?

Reply:
We can remove the `cat` in that case :+1:

Reply:
Removed in cb07afa